conference.js

```diff
@@ -2048,8 +2048,9 @@ export default {
         }
 
         APP.settings.setDisplayName(formattedNickname);
-        room.setDisplayName(formattedNickname);
-        APP.UI.changeDisplayName(this.getMyUserId(),
-            formattedNickname);
+        if (room) {
+            room.setDisplayName(formattedNickname);
+            APP.UI.changeDisplayName(this.getMyUserId(), formattedNickname);
```

Comment:
hmm the room may not be ready to broadcast the change, but shouldn't the UI event still be emitted ?